_posts/2022-11-02-relicensing.md

```diff
@@ -6,6 +6,21 @@ title: Relicensing MDAnalysis
 <img src="https://www.gnu.org/graphics/lgplv3-with-text-154x68.png"
 title="LGPLv3" alt="LGPLv3 logo" style="float: right"/>
 
+> :warning: **Disclaimer**: The MDAnalysis core team members are not
```

Comment:
todo - check if emojis are actually coming through on build

Reply:
No, they don't, I think you have to insert them. This is what the local build looks like:

![image](https://user-images.githubusercontent.com/237980/199859089-4c28c834-13cd-49a9-92ee-78a0136cc182.png)

Reply:
(Isn't the `> :warning:` syntax a GitHub specific hack?)

Reply:
https://stackoverflow.com/questions/58737436/how-to-create-a-good-looking-notification-or-warning-box-in-github-flavoured-mar

Maybe try

> **Warning**
> Abuse of block quotes in GFM!

```
> **Warning**
> Abuse of block quotes in GFM!
```

Reply:
Btw, could we start with our short intro and the LGPL logo and put the legal disclaimer at the end of the intro paragraph? I don't think that the warning is the first thing that people want to read.

Reply:
GFM warnings do not work for me when rendering locally.

Reply:
With my edits
![image](https://user-images.githubusercontent.com/237980/199862287-0ccac7a0-6919-4fcd-a7e8-96af29b26c38.png)